### DISCUSSION TITLE
Trim for subtitle file (.srt) too for a trimmed file?

### DISCUSSION THREAD

**Discussion by rieje** [mifi/lossless-cut#1965] [Q&A] [answered]
I have video subtitle files and their associated subtitle files (.srt). When I trim these videos, is there any way I can automate modifying the subtitle files so that matches the trimmed video using the trimmed info from the project file? I don't even mind a solution that bakes the subtitles into the video file if it's possible (preferably subtitles are still toggleable).

**Comment by mifi** [comment]
you can first add the subtitle to the video (first open video then drag-drop the subtitle), then export. then open the exported video and trim.

**Reply by rieje** [reply]
When I drag-drop the srt file it results in:

> Unable to playback this file. Try to convert to supported format from the menu.

When i Import Project > Subtitles (SRT), it produces hundreds of segments where each a line of subitle text and with its timestamp as the length of the segment. I'm pretty sure this is not supposed to happen because when I export it it takes a long time and stops abruptly in the process, with the file unfinished and with no subtitles baked in.

Any ideas? Also, would this result in subtitles permanently in the video or is it toggleable? I prefer the latter.

**Comment by mifi** [comment, marked as answer]
> When I drag-drop the srt file it results in:
> 
maybe you need to enable this under "settings": Ask about what to do when opening a new file when another file is already already open
